### PR TITLE
fix: remove peeringdb library and peeringdb_sync command

### PR DESCRIPTION
**Pull request by Genza999**
Implementataion here does the following:
- Remove fullctl_peeringdb_sync command from fullctl
- Remove peeringdb  library and django-peeringdb from fullctl

Task:
https://app.clickup.com/t/14289126/FC-3048

Needs https://github.com/fullctl/pdbctl/pull/19 merged

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -63,6 +60,7 @@ psycopg = {extras = ["binary"], version = ">=3.1.8"}
 # force update
 pyyaml = ">=5"
 munge = ">=1.3.0"
+django-countries = "^7.6"
```
Comment (vegu) [inline pyproject.toml:63]: @Genza999 lets do >= here

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,128 +1,168 @@
 # Changelog
 
-
```
Comment (vegu) [inline CHANGELOG.md:3]: @Genza999 what's causing these white-space changes?

File: Ctl/VERSION [hunk 3]
```diff
@@ -1 +1 @@
-1.13.0-rc.4
\ No newline at end of file
+1.13.0-rc.4
```
Comment (vegu) [inline Ctl/VERSION:1]: @Genza999 more unintentional white-space changes - this is an automatically updated / maintained file, so we don't want to make any changes to it

**Review by vegu — CHANGES_REQUESTED** [review]

**Comment by vegu** [comment]
Note that this shouldn't be merged until the required changes are merged in pdbctl (e.g., the implementation of the sync command moved to there)

**Comment by vegu** [comment]
@Genza999 i think this also had merge conflicts (plus the commit history seems off) - might be good to rebase?

File: pyproject.toml (hunk 1 above)
Comment (Genza999) [reply pyproject.toml]: This has been implemented

File: CHANGELOG.md (hunk 2 above)
Comment (Genza999) [reply CHANGELOG.md]: White space changes have been reverted!

File: Ctl/VERSION (hunk 3 above)
Comment (Genza999) [reply Ctl/VERSION]: White space/unecessary changes have been reverted

**Comment by Genza999** [comment]
> @Genza999 i think this also had merge conflicts (plus the commit history seems off) - might be good to rebase?

Conflicts with base branch have been resolved

**Review by vegu — APPROVED** [review]

**Review by vegu — CHANGES_REQUESTED** [review]
There seems to be a failing test requiring django-peeringdb still @Genza999

**Pull request merged**